.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: Lint and Test code
+
+on:
+  pull_request:
+  schedule:
+    # run once a week on early monday mornings
+    - cron: '22 2 * * 1'
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Cache pip
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements/**.txt') }}
```

Comment:
Unless we make changes to the requirements file structure, we need to change this line:

```diff
- key: ${{ runner.os }}-pip-${{ hashFiles('requirements/**.txt') }}
+ key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
```

Reply:
Good catch! Thanks